pull_request_template.md

```diff
@@ -1,38 +1,40 @@
-# Description 
-A summary of the change 
+# Description
 
+A summary of the change
+
+# Type of change
 
-# Type of change 
 - [ ] Bug fix
 - [ ] New feature
 - [ ] Refactoring
-- [ ] Documentation 
-
+- [ ] Documentation
+- [ ] Improvement
 
 # Problem
-Briefly describe the root cause and analysis of the problem 
 
+Briefly describe the root cause and analysis of the problem
 
 # Solution description
-Describe your code changes in detail for reviewers. 
+
+Describe your code changes in detail for reviewers.
 Explain the technical solution you have provided and how it fixes the issue.
 
+# Tests
 
-# Tests 
 All unit tests pass?
 
 Builds successfully?
 
+Do all SA checks pass?
 
 Attach screenshot of the expected result (e.g. UI of the website/ postman api call result/ database changes)
 
-
 Provide steps in detail for reviewers to reproduce the expected result of the code changes.
 
+# Relevant document/ link (if any)
 
-# Relevant document/ link (if any) 
-Any related documents could help reviewers to review this PR 
-
+Any related documents could help reviewers to review this PR
 
 # Risk
+
 Identity the risk level (low, medium, high) and indicate which component(s) will be affected if bugs exist.
```

Comment:
Also thinking about removing this as well since it's self explanatory when you review the code?

Reply:
IMO it could be important because someone with less context could review and approves a high-risk PR without factoring in the risk.